app.py

```diff
@@ -1,26 +1,21 @@
 class Calculator:
     
-    def __init__(self,a=10,b=10) -> None:         # -> is functional annotations
+    def __init__(self,a=10,b=10,c=10) -> None:         # -> is functional annotations
         self.a = a
         self.b = b
+        self.c = c
     
     def sum(self) -> int: 
-        return (self.a + self.b)
+        return (self.a + self.b + self.c)
     
     def diff(self) -> int:
-        if self.a > self.b:
-            return (self.a - self.b)
-        else:
-            return (self.b - self.a)
+        return (self.a - self.b - self.c)
```

Comment:
Hey emp2, during sub we should not get negative numbers. Modify code according to that.

Reply:
solved